pynecone/components/navigation/stepper.py

```diff
@@ -0,0 +1,107 @@
+"""A component to indicate progress through a multi-step process."""
+
+from pynecone.components.component import Component
+from pynecone.components.libs.chakra import ChakraComponent
+from pynecone.vars import Var
+
+
+class Stepper(ChakraComponent):
+    """The parent container for a stepper."""
+
+    tag = "Stepper"
+
+    # The color scheme to use for the stepper; default is blue
+    colorScheme: Var[str]
+
+    # chakra provides a useSteps hook to control the stepper
+    # instead, use an integer state value to set progress in the stepper
+
+    # The index of the current step
+    index: Var[int]
+
+    # The size of the steps in the stepper
+    size: Var[str]
+
+    @classmethod
+    def create(cls, *children, items=None, **props) -> Component:
```

Comment:
Let's add a type annotation for `items` because currently it's not clear what type is expected - looks like a `List[Tuple]`.